src/Message/Message.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from '../Icon'
+import styles from './Message.module.scss'
+
+const Message = ({
+  wrapperClassName,
+  className,
+  variant,
+  children,
+  icon,
+  iconClassName,
+  ...props,
+}) => (
+  <div className={cx(wrapperClassName, styles.wrapper, styles[variant])} {...props}>
+    {icon && (
+      <Icon
+        fill="inherit"
+        type={icon}
+        className={cx(iconClassName, styles.icon, styles[`icon-${variant}`])}
+      />
+    )}
+    <div>
+      {children}
+    </div>
```

Comment:
If there no logic involved with an element, why to create it?
```js
<div>
  {children}
</div>
```
change to 
```js
{children}
```

Reply:
Done.